config/locales/en.yml

```diff
@@ -153,6 +153,8 @@ en:
     new_code: Request new code
     confirm: Confirm Code
     mfa_prefs_title: Multi-Factor Authentication Preferences
+    mfa_prefs_heading: You are about to reset your multi-factor authentication
```

Comment:
this could have become it's own `mfa_warning_section` instead of appending `mfa_prefs` for each key